src/components/Button.js

```diff
@@ -24,6 +27,10 @@ const styles = StyleSheet.create({
     color: colors.lightestText,
     fontFamily: 'titillium-web-bold'
   },
+  buttonStyle: {
```

Comment:
if it is just used for landscape, it should be part of the name here, to be clear

Reply:
that's right .. !